README.rst

```diff
@@ -1,3 +1,4 @@
+This is just a test line. will be removed shortly
```

Comment:
Please remove this line

Reply:
removed already. Do I need to pull up another request after removing it?